nautobot/docs/installation/migrating-from-netbox.md

```diff
@@ -18,7 +18,7 @@ Due to a number of significant infrastructural changes between the applications,
 
 1. Export your NetBox database to a JSON file using the `manage.py dumpdata` command
 2. Install the `nautobot-netbox-importer` Nautobot plugin using `pip install nautobot-netbox-importer`
-3. Enable the `nautobot-netbox-importer` plugin in your Nautobot configuration file
+3. Enable the `nautobot_netbox_importer` plugin in your Nautobot configuration file
```

Comment:
Similar to #128 I am thinking we might just want to go ahead and mirror the docs from the readme here, or just link to the docs on how to install it, but not both.